assets/js/googlesitekit/api/middleware/preloading.test.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * createPreloadingMiddleware tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import createPreloadingMiddleware from './preloading';
+
+const preloadedData = {
+	'google-site-kit/v1/core/site/data/connection': {
+		headers: [],
+		body: {
+			connected: true,
+			resettable: true,
+			setupCompleted: true,
+		},
+	},
+
+	'google-site-kit/v1/core/user/authentication': {
+		headers: [],
+		body: {
+			authenticated: true,
+			grantedScopes: [],
+			requiredScopes: [
+				'openid',
+				'https://www.googleapis.com/auth/userinfo.profile',
+				'https://www.googleapis.com/auth/userinfo.email',
+				'https://www.googleapis.com/auth/siteverification',
+				'https://www.googleapis.com/auth/webmasters',
+				'https://www.googleapis.com/auth/adsense.readonly',
+				'https://www.googleapis.com/auth/analytics.readonly',
+			],
+			unsatisfiedScopes: [],
+		},
+	},
+};
```

Comment:
This looks like a slightly older version of the original PR than what was merged, but if you recall we opted to use generic URIs and response data rather than referencing something "real" that we use in Site Kit. You can see what we had before here: https://github.com/google/site-kit-wp/pull/1734/files#diff-028d3f0fd8e3ea4860e0f6e64db9861fL30-L41

Just to reiterate, generic references are preferred because they don't "need" to be maintained with the real code they reference, and using minimal data in supporting code makes the tests easier to read and understand.